src/MCPClient/lib/clientScripts/jsonMetadataToCSV.py

```diff
@@ -65,7 +65,9 @@ def encode_item(item):
     """
     Wraps str.encode by recursively encoding lists.
     """
-    if isinstance(item, basestring):
+    if not item:  # Handle case where json contains null.
+        return
+    elif isinstance(item, basestring):
         return item.encode('utf-8')
     else:
         return [i.encode('utf-8') for i in item]
```

Comment:
I think we should handle the same case inside this `for` loop, for `null` values inside lists. For example:

```
[
  {
    "dc.identifier": "foo",
    "filename": "objects/foo.jpg",
    "dc.subject": [
      "bar",
      null
    ]
  }
]
```